bundles/org.eclipse.equinox.http.jetty/META-INF/MANIFEST.MF

```diff
@@ -9,16 +9,16 @@ Bundle-Activator: org.eclipse.equinox.http.jetty.internal.Activator
 Import-Package: javax.servlet;version="[3.1.0,5.0.0)",
  javax.servlet.http;version="[3.1.0,5.0.0)",
  org.eclipse.equinox.http.servlet;version="[1.2.0,2.0.0)",
- org.eclipse.jetty.http;version="[12.0.1,13.0.0)",
- org.eclipse.jetty.server;version="[12.0.1,13.0.0)",
- org.eclipse.jetty.server.handler;version="[12.0.1,13.0.0)",
- org.eclipse.jetty.session;version="[12.0.1,13.0.0)",
- org.eclipse.jetty.ee8.servlet;version="[12.0.1,13.0.0)",
- org.eclipse.jetty.ee8.nested;version="[12.0.1,13.0.0)",
- org.eclipse.jetty.util;version="[12.0.1,13.0.0)",
- org.eclipse.jetty.util.component;version="[12.0.1,13.0.0)",
- org.eclipse.jetty.util.ssl;version="[12.0.1,13.0.0)",
- org.eclipse.jetty.util.thread;version="[12.0.1,13.0.0)",
+ org.eclipse.jetty.http;version="[12.1.0,13.0.0)",
```

Comment:
if jetty is not properly versioning its packages it should be

```suggestion
 org.eclipse.jetty.http;version="[12.1.0,12.2.0)",
```

this will require us to upgrade whenever a new jetty version is used and of course.